src/Symfony/Component/Security/Core/composer.json

```
@@ -26,6 +26,7 @@
         "symfony/expression-language": "^3.4|^4.0|^5.0",
         "symfony/http-foundation": "^3.4|^4.0|^5.0",
         "symfony/ldap": "^4.4|^5.0",
+        "symfony/translation": "^4.4",
```

```suggestion
        "symfony/translation": "^4.4|^5.0",
```

Is there a reason why we should exclude 5.0?